src/common/options.cc

```diff
@@ -456,7 +456,7 @@ std::vector<Option> get_global_options() {
     Option("container_image", Option::TYPE_STR, Option::LEVEL_BASIC)
     .set_description("container image (used by cephadm orchestrator)")
     .set_flag(Option::FLAG_STARTUP)
-    .set_default("docker.io/ceph/daemon-base:latest-pacific-devel"),
+    .set_default("registry.suse.com/ses/7/ceph/ceph:latest"),
```

Comment:
this one is dangerous since the current `latest` tag points to octopus, but if this is published, the `latest` tag would become pacific without, requiring users to formally perform an upgrade.

Reply:
We still need to decide where the Pacific image will end up. In other words:

  registry.suse.com/ses/7/ceph/ceph:latest
  registry.suse.com/ses/7/ceph/ceph:pacific (and not use "latest")
  registry.suse.com/ses/7p/ceph/ceph:latest

I'm sure there are other options as well.

Reply:
can we drop the `latest` tag in favor of using `v15` and `v16`?

Reply:
can we remove the `latest` tag in favor of using `v15` or `v16`?

Reply:
I'm very much in favor of that approach (with `v15` and `v16` implicitly meaning `latest` of each version).